missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/GameWorld.java

```diff
@@ -52,13 +52,14 @@ public GameWorld(Game game, String templateName) {
     }
 
     public boolean isWorld(World world) {
-        if (world == null) {
-            return false;
-        }
+        if (world == null) return false;
+
+        if ((worldName == null) || (worldName.isEmpty())) throw new IllegalArgumentException("GameWorld must be loaded first: 'gameWorld.load()'");
         return world.getName().equals(worldName);
     }
```

Comment:
Maybe an exception here is better than a ERROR in the log, because it's not a normal case.